isaacs/node-graceful-fs
# Some stat test updates

- Improve description on built-in stat constructor check
- Add async fs stats constructor test
- statSync `uid` & `gid` test fixes
  - fix test description
  - remove duplicated `Stats` constructor check
- Add async test of stat `uid` & `gid`

I noticed that the async stat call gives us different `uid` & `gid` values from the statSync call, not sure if this is relevant or not.
